documentatie/use-cases/GSB-gebruikersdoelen.md

```diff
@@ -30,20 +31,28 @@ __uitbreidingen__:
 &emsp; &emsp; &emsp; 1a2a1. De coördinator voegt het stembureau toe in de applicatie.  
 1b. De tweede invoerder heeft ook de eerste invoer gedaan:  
 1c. De invoerder selecteert het verkeerde stembureau:  
-1d. De invoerder selecteert een stembureau waar iemand anders mee bezig is:
-
-2a. De invoerder breekt de invoer af en bewaart de invoer:  
-&emsp; 2a1. Dezelfde invoerder hervat de invoer op een later moment.  
-2b. De invoerder breekt de invoer af en bewaart de invoer niet:  
-&emsp; 2a1. Dezelfde of een andere invoerder begint op een later moment opnieuw met de invoer.
-
-3a. De invoer voldoet niet aan de validatieregels voor fouten:  
-&emsp; 3a1. De applicatie toont een foutmelding voor elke gefaalde validatieregel.  
-&emsp; 3a2. [De invoerder handelt de fout(en) af.](./GSB-subfuncties.md#de-invoerder-handelt-de-fouten-af)  
-4a. De invoer voldoet niet aan de plausibiliteitschecks:  
-&emsp; 4a1. De applicatie toont een waarschuwing voor elke gefaalde plausibiliteitscheck.  
+1d. De invoerder selecteert een stembureau waar iemand anders mee bezig is:  
+1e. De invoerder selecteert een stembureau waar een andere gebruiker invoer voor heeft opgeslagen:  
+1f. De invoerder selecteert een stembureau waarvan de invoer al definitief is:
```

Comment:
Deze staan niet in de lijst van stembureaus.

Reply:
Maar je kunt wel het nummer invullen en dan op "Beginnen" klikken.

Dus: uitbreiding aangevuld met dat de applicatie een foutmelding toont.